src/lib/BytesHelper.ts

```diff
@@ -37,6 +37,18 @@ export function getAssetId(
   return `0x${result.toString(16).padStart(64, '0')}`;
 }
 
+export function encode(address: string, callData: string): string {
+  const encodedAddress = ethers.utils.defaultAbiCoder.encode(
```

Comment:
Where do you import `ethers`? I thought this repo used `web3` as a dependency instead of `ethers`?